src/ShapeCrawler/Drawing/Picture.cs

```diff
@@ -135,7 +145,45 @@ internal override void CopyTo(
         var copy = this.PShapeTreeElement.CloneNode(true);
         copy.Descendants<A.Blip>().First().Embed = targetImagePartRId;
     }
-    
+
+    /// <summary>
+    ///     Set the cropping frame values onto the supplied source rectangle.
+    /// </summary>
+    /// <param name="frame">Source values to get cropping values.</param>
+    /// <param name="aSrcRect">Rectangle to be updated with our values.</param>
+    private static void ApplyCropToSourceRectangle(CroppingFrame frame, A.SourceRectangle aSrcRect)
+    {
+        aSrcRect.Left = ToThousandths(frame.left);
+        aSrcRect.Right = ToThousandths(frame.right);
+        aSrcRect.Top = ToThousandths(frame.top);
+        aSrcRect.Bottom = ToThousandths(frame.bottom);        
+    }
+
+    /// <summary>
+    ///     Convert a source rectangle to a cropping frame.
+    /// </summary>
+    /// <param name="aSrcRect">Source rectangle which contains the needed frame.</param>
+    /// <returns>Resulting frame.</returns>
+    private static CroppingFrame CroppingFrameFromSourceRectangle(A.SourceRectangle? aSrcRect)
+    {
+        if (aSrcRect is null)
+        {
+            return new CroppingFrame(0,0,0,0);
+        }
+
+        return new CroppingFrame(
+            FromThousandths(aSrcRect.Left),
+            FromThousandths(aSrcRect.Right),
+            FromThousandths(aSrcRect.Top),
+            FromThousandths(aSrcRect.Bottom));
+    }
+
+    private static decimal FromThousandths(Int32Value? int32) => 
+        int32 is not null ? int32 / 1000m : 0;
```

Comment:
It would be good to somehow downgrade the magic of that number `1000`. For example, with comment `// the original value is scaled by 1000 to achieve higher precision`.

Reply:
2184c2bfc2e66bdaa15d76eb2c337560a7be7c55